[BUG] Flaky test failure KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip

### Details
As seen in the [build failure](https://github.com/opensearch-project/search-processor/actions/runs/4287151291/jobs/7467656367) in #106, the `testSerializeToXContentRoundtrip` test in [KendraIntelligentRankingConfigurationTests](https://github.com/opensearch-project/search-processor/blob/82e59f1fd8ed653598f2d7127734f4666b0f1549/src/test/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfigurationTests.java#L27) fails sporadically.

### Steps to reproduce
The following commands include the random seed to reproduce the error.
```
search-processor (82e59f1) ✔ ./gradlew ':test' --tests "org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip" -Dtests.seed=C94C719B5F62ADE -Dtests.security.manager=true -Dtests.locale=vi-VN -Dtests.timezone=Etc/GMT+9                
=======================================
OpenSearch Build Hamster says Hello!
  Gradle Version        : 7.4.2
  OS Info               : Mac OS X 12.6.3 (aarch64)
  JDK Version           : 11 (Eclipse Temurin JDK)
  JAVA_HOME             : /Library/Java/JavaVirtualMachines/jdk-11.0.17+8/Contents/Home
  Random Testing Seed   : C94C719B5F62ADE
  In FIPS 140 mode      : false
=======================================

> Task :test FAILED


org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests > testSerializeToXContentRoundtrip FAILED
    ParsingException[Failed to parse value [0] for Transformer order, must be >= 1]
        at __randomizedtesting.SeedInfo.seed([C94C719B5F62ADE:1195B20D26FE1F21]:0)
        at app//org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfiguration.parse(KendraIntelligentRankingConfiguration.java:84)
        at app//org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip(KendraIntelligentRankingConfigurationTests.java:36)
REPRODUCE WITH: ./gradlew ':test' --tests "org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip" -Dtests.seed=C94C719B5F62ADE -Dtests.security.manager=true -Dtests.locale=vi-VN -Dtests.timezone=Etc/GMT+9 -Druntime.java=11

Suite: Test class org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests
  1> [2023-02-27T17:08:19,270][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[C94C719B5F62ADE]]] unable to load JNA native support library, native methods will be disabled.
  1> java.lang.UnsatisfiedLinkError: Can't load library: /Users/lnse/Library/Caches/JNA/temp/jna10477499904397165298.tmp
  1>    at java.lang.ClassLoader.loadLibrary(ClassLoader.java:2633) ~[?:?]
  1>    at java.lang.Runtime.load0(Runtime.java:768) ~[?:?]
  1>    at java.lang.System.load(System.java:1837) ~[?:?]
  1>    at com.sun.jna.Native.loadNativeDispatchLibraryFromClasspath(Native.java:1018) ~[jna-5.5.0.jar:5.5.0 (b0)]
  1>    at com.sun.jna.Native.loadNativeDispatchLibrary(Native.java:988) ~[jna-5.5.0.jar:5.5.0 (b0)]
  1>    at com.sun.jna.Native.<clinit>(Native.java:195) ~[jna-5.5.0.jar:5.5.0 (b0)]
  1>    at java.lang.Class.forName0(Native Method) ~[?:?]
  1>    at java.lang.Class.forName(Class.java:315) ~[?:?]
  1>    at org.opensearch.bootstrap.Natives.<clinit>(Natives.java:60) [opensearch-2.5.0.jar:2.5.0]
  1>    at org.opensearch.bootstrap.Bootstrap.initializeNatives(Bootstrap.java:123) [opensearch-2.5.0.jar:2.5.0]
  1>    at org.opensearch.bootstrap.BootstrapForTesting.<clinit>(BootstrapForTesting.java:105) [framework-2.5.0.jar:2.5.0]
  1>    at org.opensearch.test.OpenSearchTestCase.<clinit>(OpenSearchTestCase.java:257) [framework-2.5.0.jar:2.5.0]
  1>    at java.lang.Class.forName0(Native Method) ~[?:?]
  1>    at java.lang.Class.forName(Class.java:398) [?:?]
  1>    at com.carrotsearch.randomizedtesting.RandomizedRunner$2.run(RandomizedRunner.java:623) [randomizedtesting-runner-2.7.1.jar:?]
  1> [2023-02-27T17:08:19,285][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[C94C719B5F62ADE]]] cannot check if running as root because JNA is not available
  1> [2023-02-27T17:08:19,285][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[C94C719B5F62ADE]]] cannot install system call filter because JNA is not available
  1> [2023-02-27T17:08:19,286][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[C94C719B5F62ADE]]] cannot register console handler because JNA is not available
  1> [2023-02-27T17:08:19,287][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[C94C719B5F62ADE]]] cannot getrlimit RLIMIT_NPROC because JNA is not available
  1> [2023-02-27T17:08:19,287][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[C94C719B5F62ADE]]] cannot getrlimit RLIMIT_AS because JNA is not available
  1> [2023-02-27T17:08:19,287][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[C94C719B5F62ADE]]] cannot getrlimit RLIMIT_FSIZE because JNA is not available
  1> [2023-02-27T16:08:19,665][INFO ][o.o.s.r.t.k.c.KendraIntelligentRankingConfigurationTests] [testSerializeToXContentRoundtrip] before test
  1> [2023-02-27T16:08:19,868][INFO ][o.o.s.r.t.k.c.KendraIntelligentRankingConfigurationTests] [testSerializeToXContentRoundtrip] after test
  2> REPRODUCE WITH: ./gradlew ':test' --tests "org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip" -Dtests.seed=C94C719B5F62ADE -Dtests.security.manager=true -Dtests.locale=vi-VN -Dtests.timezone=Etc/GMT+9 -Druntime.java=11
  2> ParsingException[Failed to parse value [0] for Transformer order, must be >= 1]
        at __randomizedtesting.SeedInfo.seed([C94C719B5F62ADE:1195B20D26FE1F21]:0)
        at app//org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfiguration.parse(KendraIntelligentRankingConfiguration.java:84)
        at app//org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip(KendraIntelligentRankingConfigurationTests.java:36)
  2> NOTE: leaving temporary files on disk at: /Users/lnse/opensearch/search-processor/build/testrun/test/temp/org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests_C94C719B5F62ADE-002
  2> NOTE: test params are: codec=Asserting(Lucene94): {}, docValues:{}, maxPointsInLeafNode=1933, maxMBSortInHeap=6.854586443954316, sim=Asserting(RandomSimilarity(queryNorm=true): {}), locale=vi-VN, timezone=Etc/GMT+9
  2> NOTE: Mac OS X 12.6.3 aarch64/Eclipse Adoptium 11.0.17 (64-bit)/cpus=10,threads=1,free=405602816,total=536870912
  2> NOTE: All tests run in this JVM: [KendraIntelligentRankingConfigurationTests]

Tests with failures:
 - org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip

1 test completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///Users/lnse/opensearch/search-processor/build/reports/tests/test/index.html

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.4.2/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 2s
7 actionable tasks: 2 executed, 5 up-to-date
```


```
 search-processor (main) ✔ ./gradlew ':test' --tests "org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip" -Dtests.seed=7072CFFCEB4B3DC0 -Dtests.security.manager=true -Dtests.locale=mk -Dtests.timezone=Europe/Madrid
=======================================
OpenSearch Build Hamster says Hello!
  Gradle Version        : 7.4.2
  OS Info               : Mac OS X 12.6.3 (aarch64)
  JDK Version           : 11 (Eclipse Temurin JDK)
  JAVA_HOME             : /Library/Java/JavaVirtualMachines/jdk-11.0.17+8/Contents/Home
  Random Testing Seed   : 7072CFFCEB4B3DC0
  In FIPS 140 mode      : false
=======================================

> Task :test FAILED


org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests > testSerializeToXContentRoundtrip FAILED
    ParsingException[Failed to parse value [0] for Transformer order, must be >= 1]
        at __randomizedtesting.SeedInfo.seed([7072CFFCEB4B3DC0:6D73BAE87843083F]:0)
        at app//org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfiguration.parse(KendraIntelligentRankingConfiguration.java:84)
        at app//org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip(KendraIntelligentRankingConfigurationTests.java:36)
REPRODUCE WITH: ./gradlew ':test' --tests "org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip" -Dtests.seed=7072CFFCEB4B3DC0 -Dtests.security.manager=true -Dtests.locale=mk -Dtests.timezone=Europe/Madrid -Druntime.java=11

Suite: Test class org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests
  1> [2023-02-28T12:29:02,445][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[7072CFFCEB4B3DC0]]] unable to load JNA native support library, native methods will be disabled.
  1> java.lang.UnsatisfiedLinkError: Can't load library: /Users/lnse/Library/Caches/JNA/temp/jna8244722722494699276.tmp
  1>    at java.lang.ClassLoader.loadLibrary(ClassLoader.java:2633) ~[?:?]
  1>    at java.lang.Runtime.load0(Runtime.java:768) ~[?:?]
  1>    at java.lang.System.load(System.java:1837) ~[?:?]
  1>    at com.sun.jna.Native.loadNativeDispatchLibraryFromClasspath(Native.java:1018) ~[jna-5.5.0.jar:5.5.0 (b0)]
  1>    at com.sun.jna.Native.loadNativeDispatchLibrary(Native.java:988) ~[jna-5.5.0.jar:5.5.0 (b0)]
  1>    at com.sun.jna.Native.<clinit>(Native.java:195) ~[jna-5.5.0.jar:5.5.0 (b0)]
  1>    at java.lang.Class.forName0(Native Method) ~[?:?]
  1>    at java.lang.Class.forName(Class.java:315) ~[?:?]
  1>    at org.opensearch.bootstrap.Natives.<clinit>(Natives.java:60) [opensearch-2.5.0.jar:2.5.0]
  1>    at org.opensearch.bootstrap.Bootstrap.initializeNatives(Bootstrap.java:123) [opensearch-2.5.0.jar:2.5.0]
  1>    at org.opensearch.bootstrap.BootstrapForTesting.<clinit>(BootstrapForTesting.java:105) [framework-2.5.0.jar:2.5.0]
  1>    at org.opensearch.test.OpenSearchTestCase.<clinit>(OpenSearchTestCase.java:257) [framework-2.5.0.jar:2.5.0]
  1>    at java.lang.Class.forName0(Native Method) ~[?:?]
  1>    at java.lang.Class.forName(Class.java:398) [?:?]
  1>    at com.carrotsearch.randomizedtesting.RandomizedRunner$2.run(RandomizedRunner.java:623) [randomizedtesting-runner-2.7.1.jar:?]
  1> [2023-02-28T12:29:02,457][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[7072CFFCEB4B3DC0]]] cannot check if running as root because JNA is not available
  1> [2023-02-28T12:29:02,458][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[7072CFFCEB4B3DC0]]] cannot install system call filter because JNA is not available
  1> [2023-02-28T12:29:02,458][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[7072CFFCEB4B3DC0]]] cannot register console handler because JNA is not available
  1> [2023-02-28T12:29:02,459][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[7072CFFCEB4B3DC0]]] cannot getrlimit RLIMIT_NPROC because JNA is not available
  1> [2023-02-28T12:29:02,459][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[7072CFFCEB4B3DC0]]] cannot getrlimit RLIMIT_AS because JNA is not available
  1> [2023-02-28T12:29:02,459][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[7072CFFCEB4B3DC0]]] cannot getrlimit RLIMIT_FSIZE because JNA is not available
  1> [2023-02-28T21:29:02,832][INFO ][o.o.s.r.t.k.c.KendraIntelligentRankingConfigurationTests] [testSerializeToXContentRoundtrip] before test
  1> [2023-02-28T21:29:03,063][INFO ][o.o.s.r.t.k.c.KendraIntelligentRankingConfigurationTests] [testSerializeToXContentRoundtrip] after test
  2> REPRODUCE WITH: ./gradlew ':test' --tests "org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip" -Dtests.seed=7072CFFCEB4B3DC0 -Dtests.security.manager=true -Dtests.locale=mk -Dtests.timezone=Europe/Madrid -Druntime.java=11
  2> ParsingException[Failed to parse value [0] for Transformer order, must be >= 1]
        at __randomizedtesting.SeedInfo.seed([7072CFFCEB4B3DC0:6D73BAE87843083F]:0)
        at app//org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfiguration.parse(KendraIntelligentRankingConfiguration.java:84)
        at app//org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip(KendraIntelligentRankingConfigurationTests.java:36)
  2> NOTE: leaving temporary files on disk at: /Users/lnse/opensearch/search-processor/build/testrun/test/temp/org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests_7072CFFCEB4B3DC0-001
  2> NOTE: test params are: codec=Asserting(Lucene94): {}, docValues:{}, maxPointsInLeafNode=863, maxMBSortInHeap=6.6242684347487675, sim=Asserting(RandomSimilarity(queryNorm=false): {}), locale=mk, timezone=Europe/Madrid
  2> NOTE: Mac OS X 12.6.3 aarch64/Eclipse Adoptium 11.0.17 (64-bit)/cpus=10,threads=1,free=406309992,total=536870912
  2> NOTE: All tests run in this JVM: [KendraIntelligentRankingConfigurationTests]

Tests with failures:
 - org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip

1 test completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///Users/lnse/opensearch/search-processor/build/reports/tests/test/index.html

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.4.2/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 2s
7 actionable tasks: 1 executed, 6 up-to-date
```


```
search-processor (main) ✔ ./gradlew ':test' --tests "org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip" -Dtests.seed=5651DD3B7CACCD87 -Dtests.security.manager=true -Dtests.locale=no -Dtests.timezone=Etc/GMT-9
=======================================
OpenSearch Build Hamster says Hello!
  Gradle Version        : 7.4.2
  OS Info               : Mac OS X 12.6.3 (aarch64)
  JDK Version           : 11 (Eclipse Temurin JDK)
  JAVA_HOME             : /Library/Java/JavaVirtualMachines/jdk-11.0.17+8/Contents/Home
  Random Testing Seed   : 5651DD3B7CACCD87
  In FIPS 140 mode      : false
=======================================

> Task :test FAILED


org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests > testSerializeToXContentRoundtrip FAILED
    ParsingException[Failed to parse value [0] for Transformer order, must be >= 1]
        at __randomizedtesting.SeedInfo.seed([5651DD3B7CACCD87:4B50A82FEFA4F878]:0)
        at app//org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfiguration.parse(KendraIntelligentRankingConfiguration.java:84)
        at app//org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip(KendraIntelligentRankingConfigurationTests.java:36)
REPRODUCE WITH: ./gradlew ':test' --tests "org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip" -Dtests.seed=5651DD3B7CACCD87 -Dtests.security.manager=true -Dtests.locale=no -Dtests.timezone=Etc/GMT-9 -Druntime.java=11

Suite: Test class org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests
  1> [2023-02-28T12:30:37,520][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[5651DD3B7CACCD87]]] unable to load JNA native support library, native methods will be disabled.
  1> java.lang.UnsatisfiedLinkError: Can't load library: /Users/lnse/Library/Caches/JNA/temp/jna6176494812616682527.tmp
  1>    at java.lang.ClassLoader.loadLibrary(ClassLoader.java:2633) ~[?:?]
  1>    at java.lang.Runtime.load0(Runtime.java:768) ~[?:?]
  1>    at java.lang.System.load(System.java:1837) ~[?:?]
  1>    at com.sun.jna.Native.loadNativeDispatchLibraryFromClasspath(Native.java:1018) ~[jna-5.5.0.jar:5.5.0 (b0)]
  1>    at com.sun.jna.Native.loadNativeDispatchLibrary(Native.java:988) ~[jna-5.5.0.jar:5.5.0 (b0)]
  1>    at com.sun.jna.Native.<clinit>(Native.java:195) ~[jna-5.5.0.jar:5.5.0 (b0)]
  1>    at java.lang.Class.forName0(Native Method) ~[?:?]
  1>    at java.lang.Class.forName(Class.java:315) ~[?:?]
  1>    at org.opensearch.bootstrap.Natives.<clinit>(Natives.java:60) [opensearch-2.5.0.jar:2.5.0]
  1>    at org.opensearch.bootstrap.Bootstrap.initializeNatives(Bootstrap.java:123) [opensearch-2.5.0.jar:2.5.0]
  1>    at org.opensearch.bootstrap.BootstrapForTesting.<clinit>(BootstrapForTesting.java:105) [framework-2.5.0.jar:2.5.0]
  1>    at org.opensearch.test.OpenSearchTestCase.<clinit>(OpenSearchTestCase.java:257) [framework-2.5.0.jar:2.5.0]
  1>    at java.lang.Class.forName0(Native Method) ~[?:?]
  1>    at java.lang.Class.forName(Class.java:398) [?:?]
  1>    at com.carrotsearch.randomizedtesting.RandomizedRunner$2.run(RandomizedRunner.java:623) [randomizedtesting-runner-2.7.1.jar:?]
  1> [2023-02-28T12:30:37,532][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[5651DD3B7CACCD87]]] cannot check if running as root because JNA is not available
  1> [2023-02-28T12:30:37,532][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[5651DD3B7CACCD87]]] cannot install system call filter because JNA is not available
  1> [2023-02-28T12:30:37,533][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[5651DD3B7CACCD87]]] cannot register console handler because JNA is not available
  1> [2023-02-28T12:30:37,534][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[5651DD3B7CACCD87]]] cannot getrlimit RLIMIT_NPROC because JNA is not available
  1> [2023-02-28T12:30:37,534][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[5651DD3B7CACCD87]]] cannot getrlimit RLIMIT_AS because JNA is not available
  1> [2023-02-28T12:30:37,534][WARN ][o.o.b.Natives            ] [[SUITE-KendraIntelligentRankingConfigurationTests-seed#[5651DD3B7CACCD87]]] cannot getrlimit RLIMIT_FSIZE because JNA is not available
  1> [2023-03-01T05:30:37,887][INFO ][o.o.s.r.t.k.c.KendraIntelligentRankingConfigurationTests] [testSerializeToXContentRoundtrip] before test
  1> [2023-03-01T05:30:38,107][INFO ][o.o.s.r.t.k.c.KendraIntelligentRankingConfigurationTests] [testSerializeToXContentRoundtrip] after test
  2> REPRODUCE WITH: ./gradlew ':test' --tests "org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip" -Dtests.seed=5651DD3B7CACCD87 -Dtests.security.manager=true -Dtests.locale=no -Dtests.timezone=Etc/GMT-9 -Druntime.java=11
  2> ParsingException[Failed to parse value [0] for Transformer order, must be >= 1]
        at __randomizedtesting.SeedInfo.seed([5651DD3B7CACCD87:4B50A82FEFA4F878]:0)
        at app//org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfiguration.parse(KendraIntelligentRankingConfiguration.java:84)
        at app//org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip(KendraIntelligentRankingConfigurationTests.java:36)
  2> NOTE: leaving temporary files on disk at: /Users/lnse/opensearch/search-processor/build/testrun/test/temp/org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests_5651DD3B7CACCD87-001
  2> NOTE: test params are: codec=Asserting(Lucene94): {}, docValues:{}, maxPointsInLeafNode=1670, maxMBSortInHeap=5.474299831647895, sim=Asserting(RandomSimilarity(queryNorm=true): {}), locale=no, timezone=Etc/GMT-9
  2> NOTE: Mac OS X 12.6.3 aarch64/Eclipse Adoptium 11.0.17 (64-bit)/cpus=10,threads=1,free=402628568,total=536870912
  2> NOTE: All tests run in this JVM: [KendraIntelligentRankingConfigurationTests]

Tests with failures:
 - org.opensearch.search.relevance.transformer.kendraintelligentranking.configuration.KendraIntelligentRankingConfigurationTests.testSerializeToXContentRoundtrip

1 test completed, 1 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///Users/lnse/opensearch/search-processor/build/reports/tests/test/index.html

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.4.2/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 2s
7 actionable tasks: 1 executed, 6 up-to-date
```

--- a/src/test/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfigurationTests.java
+++ b/src/test/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/configuration/KendraIntelligentRankingConfigurationTests.java
@@ -48,7 +48,7 @@ public class KendraIntelligentRankingConfigurationTests extends OpenSearchTestCa
     }
 
     private static KendraIntelligentRankingConfiguration getKendraIntelligentRankingConfiguration() {
-        int order = randomInt(10);
+        int order = randomInt(10) + 1;
         int docLimit = randomInt( Integer.MAX_VALUE - 25) + 25;
         KendraIntelligentRankingConfiguration.KendraIntelligentRankingProperties properties =
                 new KendraIntelligentRankingConfiguration.KendraIntelligentRankingProperties(List.of("body1"),

